distribution/docs/src/main/resources/content/_configuring/reusing-configurations.adoc

```diff
@@ -17,7 +17,7 @@ To export the current configuration settings:
 To import previously exported configuration settings:
 
 . Install ${branding}
-. Restore all external files, softlinks, and directories that would not have been exported and for which warnings would have been generated during export. This could include among other things: external certificates, content store, and monitored directories.
+. Restore all external files, softlinks, and directories that would not have been exported and for which warnings would have been generated during export. This could include among other things such as external certificates or monitored directories.
```

Comment:
🖊 Using "among other things" and "such as" in the same sentence sounds odd.

Reply:
@lessarderic This was a wording change requested on my PR